client/mass/nav.js

```diff
@@ -624,6 +626,20 @@ function setRenderers(self) {
 				.text(selectCohort.asterisk)
 		}
 	}
+
+	self.showReleaseVersion = appState => {
+		if ((!appState?.termdbConfig?.selectCohort && !appState.termdbConfig?.massNav?.tabs?.about) || self.pkgver == null)
```

Comment:
A subjective preference, but I'd prefer showing the version somewhere in the top right corner of the subheader. Or, maybe have a `Version` button/text or info icon next to the session button, that can be clicked to show a menu with version info.

Reply:
for a ds without subcohort, it should always have the About tab.

Reply:
Ok, do you want that complete in this PR or in a separate one? Also if there's not other information besides the release version, should the about tab be the active tab or should the charts tab show by default?

Reply:
1. no worry. add persistent About tab in later PR.
2. if no other info, the About tab just show dslabel and release version
3. About tab should be the default active tab